packages/client/src/object/aggregateOrThrow.test.ts

```diff
@@ -165,16 +165,16 @@ describe("aggregateOrThrow", () => {
       },
       {
         select: {
-          text: "approximateDistinct",
+          id: "approximateDistinct",
         },
         groupBy: {
           text: "exact",
         },
       },
     );
     expectType<Array<any>>(grouped);
-    expectType<string | undefined>(grouped[0].group.text);
-    expectType<number>(grouped[0].values.text.approximateDistinct);
+    expectType<string | undefined>(grouped[0].$group?.text);
```

Comment:
I had to optional chain here to get the tests to pass, but I have no idea why my changes did that

Reply:
For context ,the error tests gave was `cannot read properties of undefined` on group and id, which is why I optional chained it

Reply:
Actually, was able to fix this